swupd/create_manifests.go

```diff
@@ -295,6 +293,35 @@ func CreateManifests(version uint32, minVersion uint32, format uint, statedir st
 		}
 	}
 
+	allManifests := newManifests
+	// add all manifests in MoM, including old ones, to the list to index for swupd search
+	for _, m := range newMoM.Files {
+		if m.Version < version {
```

Comment:
Your comment above doesn't make sense, this only adds old manifests to the list...?

Reply:
`allManifests := newManifests` above, then I append all old ones. I'll clarify the comment.

Reply:
Done